examples/e-commerce/package.json

```diff
@@ -16,10 +16,10 @@
     "@angular/platform-browser": "12.0.3",
     "@angular/platform-browser-dynamic": "12.0.3",
     "@angular/router": "12.0.3",
-    "algoliasearch": "4.4.0",
-    "angular-instantsearch": "3.0.0-beta.1",
+    "algoliasearch": "4.10.3",
+    "angular-instantsearch": "file:../../dist",
```

Comment:
does this have an impact for the e2e test?

Reply:
It doesn't look like e2e tests are running on angular examples. So I guess the answer is no 😄 .
We'll fix this at a later point.

The examples run fine however: https://deploy-preview-821--angular-instantsearch.netlify.app/examples/e-commerce/
And it's much easier to test the current commit of the lib this way. Not sure why we haven't done it earlier.